### ISSUE TITLE
Output format should be table by default

### ISSUE THREAD

**Issue by lorenzotomasini** [openbaton/openbaton-cli#8]
Without asking which one to use

**Comment by lorenzotomasini** [comment]
and when choosing json the full **valid** json should be returned:

```
(openbaton-cli) openbaton-cli|master⚡ ⇒ ../openbaton nsr list
insert format: json

[u'89a353ef-47d6-40db-b949-883c099aeecf', u'NSD iperf + floatingIPs', u'ACTIVE', u'Scaled in', u'FOKUS', u'1.0']```

**Comment by ananyajh** [comment]
Fixed by Pull Request #10
